Makefile

```diff
@@ -28,9 +28,7 @@ testcoverage:
 	pytest woodwork/ --cov=woodwork
 
 .PHONY: installdeps
-installdeps:
-	pip install --upgrade pip
-	pip install -e .
```

Comment:
Why was this removed? Shouldn't we expand this to cover
`pip install -e .`, `pip install -e ".[dev]"` and `pip install -e ".[test]"`

Reply:
Additionally, we don't use the test, and dev commands anywhere (CI wise). Devs can just directly install them via `pip`.